com.unity.multiplayer.mlapi/Runtime/Core/RPCQueue/RPCQueueContainer.cs

```diff
@@ -151,19 +150,19 @@ public void AdvanceFrameHistory(QueueHistoryFrame.QueueFrameType queueType)
 
             if (!QueueHistory.ContainsKey(queueType))
             {
-                UnityEngine.Debug.LogError("You must initialize the RPCQueueManager before using MLAPI!");
+                UnityEngine.Debug.LogError("You must initialize the RpcQueueContainer before using MLAPI!");
```

Comment:
put `using UnityEngine;` at top and replace `UnityEngine.Debug...` with `Debug...` only?